1.0/docs/devguide/properties.md

```diff
@@ -422,70 +422,42 @@ Example:
       </script>
     </dom-module>
 
-### Deep sub-property observation {#deep-observation}
-
-To call an observer when any (deep) sub-property of an
-object changes, specify a path with a wildcard (`*`).
-
-When you specify a path with a wildcard, the argument passed to your
-observer is a change record object with the following properties:
+### Observe array mutations
 
-*   `path`. Path to the property that changed. 
-*   `value`. New value of the path that changed.
-*   `base`. The object matching the non-wildcard portion of the path. 
+Use a splice observer to call a method whenever an array item is added
+or deleted via `push`, `pop`, `shift`, `unshift`, or `splice`. Note that 
+these methods are collectively referred to as **splices**.
```

Comment:
I think this last sentence isn't quite right. The original text that I wrote was also not quite right. Here's the background:

A splice operation (as performed by the `splice` method) removes zero or more elements from a given point in the array, then inserting zero or more new elements in the same location. So (and here's the crucial bit) all of these array mutation methods can be _represented_ in terms of splices.

So, `arr.push(item)` is equivalent to `arr.splice(arr.length,0,item)`. `arr.pop` is equivalent to `arr.splice(arr.length-1,1)`, `arr.unshift(item)` is equivalent to `arr.splice(0,0,item)`, and so on. 

Maybe, "Whenever the array is mutated, the observer receives a change record representing the mutation as a set of array splices." Whew. That's a mouthful, but it's probably more correct.